joern-cli/frontends/x2cpg/src/main/scala/io/joern/x2cpg/utils/dependency/GradleDependencies.scala

```diff
@@ -201,31 +188,30 @@ object GradleDependencies {
       }
 
     if (connectionOption.isDefined) {
-      val connection = connectionOption.get
+      val connection   = connectionOption.get
+      val stdoutStream = new ByteArrayOutputStream
+      val stderrStream = new ByteArrayOutputStream
       try {
-        logger.info(s"Executing gradle task '$taskName'...")
+        logger.info(s"Executing gradle task '${initScript.taskName}'...")
         connection
           .newBuild()
-          .forTasks(taskName)
-          // .setStandardOutput(System.out) // uncomment for debugging
-          // .setStandardError(System.err)  // uncomment for debugging
+          .forTasks(initScript.taskName)
+          .withArguments("--init-script", initScriptFile.pathAsString)
+          .setStandardOutput(stdoutStream)
+          .setStandardError(stderrStream)
           .run()
       } catch {
         case t: Throwable =>
-          logger.warn(s"Caught exception while executing Gradle task: '${t.getMessage}'.")
+          logger.warn(s"Caught exception while executing Gradle task: '${t.getMessage}'")
```

Comment:
maybe you wanna pass the exception to the logger as a second param as well? maybe not... just saying